Cargo.toml

```
@@ -30,8 +30,8 @@ anyhow = "1.0"
 askalono = "0.4.3"
 byte-unit = "4.0.12"
 bytecount = "0.6.2"
-chrono = "0.4"
-chrono-humanize = "0.2.1"
+time-humanize = {version="0.1.2", features = ["time"]}
+time = { version = "0.3.5", features = ["formatting"]}
```

```suggestion
time-humanize = { version="0.1.2", features = ["time"] }
time = { version = "0.3.5", features = ["formatting"] }
```
Minor nitpick. I don't think we have a preferred style, but I've usually seen spaces inside braces.
For example, [the docs for dependencies in `Cargo.toml`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-other-registries).

---

Actually ignore the above nitpick, since my suggestion doesn't match the rest of `Cargo.toml` :sweat_smile: 
If anything, it would be more like
```suggestion
time-humanize = {version="0.1.2", features = ["time"]}
time = {version = "0.3.5", features = ["formatting"]}
```